### PR TITLE
Add outstanding chains with Alchemy support

### PR DESCRIPTION
**Pull request by sdlyy**
Resolves L2B-2529

Todos:
- [x] Re-check chain configurations
- [ ] Push envs for working chains to heroku
- [x] Polish structs responsible for missing mulitisig
- [x] .env based config for local and prod/stag can be safely shared since we dupe those in each file for no reason

### CODE REVIEW

File: packages/frontend/src/view/components/LayerZeroMultisig.tsx [hunk 1]
```diff
@@ -6,29 +6,45 @@ interface LzMultisigProps {
...
-        <pre className="grow overflow-auto border border-gray-700 bg-gray-900 p-6 font-mono text-gray-500">
-          {JSON.stringify(props.owners, null, 2)}
-        </pre>
+        {address ? (
+          <span className="font-mono text-gray-600">{address}</span>
+        ) : (
+          <span className="font-mono text-gray-600">
+            This chain does not support LayerZero Multisig
+          </span>
+        )}
```
Comment (michalsidzej) [inline packages/frontend/src/view/components/LayerZeroMultisig.tsx:29]: I'm proposing 2 solutions when the LZ Multisig is not present: 
1. (preferred) - Change the name to "Contracts Owner" and mark it as <problematic (design to be defined)>. We can say that GnosisSafe is not supported on this chain, but only if we're sure about it (Gnosis API?)
2. Just do not display this component, we will mark it <problematic> next to the `owner` contract value field both on ULNv2 and Endpoint

File: packages/backend/src/config/discoveryConfig.ts [hunk 2]
```diff
@@ -42,6 +49,7 @@ function createConfigFromTemplate(
           },
         },
       },
+      // Maybe we should remove that if there is no multisig support?
       'LayerZero Multisig': {
         ignoreInWatchMode: ['nonce'],
       },
```
Comment (michalsidzej) [inline packages/backend/src/config/discoveryConfig.ts:55]: Removing the override is not necessary as Discovery does not check if all the overrides were used.

File: packages/backend/src/config/config.common.ts [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+import { Config, EthereumLikeDiscoveryConfig } from './Config'
+import { arbitrumDiscoveryConfig } from './discovery/arbitrum'
+import { baseDiscoveryConfig } from './discovery/base'
+import { ethereumDiscoveryConfig } from './discovery/ethereum'
+import { optimismDiscoveryConfig } from './discovery/optimism'
+import { polygonPosDiscoveryConfig } from './discovery/polygon-pos'
+import { polygonZkEvmDiscoveryConfig } from './discovery/polygon-zkevm'
+
+export function getCommonDiscoveryConfig(env: Env): Config['discovery'] {
```
Comment (michalsidzej) [inline packages/backend/src/config/config.common.ts:14]: Amazing

**Review by michalsidzej — APPROVED** [review]
LGTM

File: packages/frontend/src/view/components/LayerZeroMultisig.tsx (hunk 1 above)
Comment (sdlyy) [reply packages/frontend/src/view/components/LayerZeroMultisig.tsx:29]: I hope you have nothing against doing that as a separate task and merging this one as-is. There is much more going behind the scenes, and I would end-up creating temporary fix ups either way. Let's discuss best approach here async and do the actual implementation separately

**Comment by sdlyy** [comment]
Envs are up

**Pull request merged**